app/components/report/ReportBuilder/ReportContainer/Content/index.tsx

```diff
@@ -0,0 +1,205 @@
+import React from 'react';
+import { _cs } from '@togglecorp/fujs';
+import { removeNull } from '@togglecorp/toggle-form';
+import {
+    Message,
+    Kraken,
+} from '@the-deep/deep-ui';
+import ReactMarkdown from 'react-markdown';
+
+import {
+    AnalysisReportContainerContentTypeEnum,
+    AnalysisReportContainerContentConfigurationType,
+} from '#generated/types';
+
+import {
+    type ContentDataType,
+    type ConfigType,
+} from '../../../schema';
+import {
+    resolveTextStyle,
+    type ContentDataFileMap,
+} from '../../../utils';
+
+import styles from './styles.css';
+
+interface Props {
+    contentType: AnalysisReportContainerContentTypeEnum;
+    contentData: ContentDataType[] | undefined;
+    configuration: AnalysisReportContainerContentConfigurationType | undefined;
+    generalConfiguration: ConfigType | undefined;
+    contentDataToFileMap: ContentDataFileMap | undefined;
+}
+
+function Content(props: Props) {
+    const {
+        contentType,
+        configuration: configurationFromProps = {},
+        generalConfiguration,
+        contentData,
+        contentDataToFileMap,
+    } = props;
+
+    const configuration = removeNull(configurationFromProps);
+
+    if (contentType === 'HEADING') {
+        const {
+            heading: {
+                content,
+                variant,
+                style,
+            } = {},
+        } = configuration;
+
+        if (variant === 'H4') {
+            return (
+                <h4
+                    className={_cs(
+                        styles.heading,
+                        styles.headingFour,
+                    )}
+                    style={resolveTextStyle(
+                        style?.content,
+                        generalConfiguration?.headingContentStyle?.h4,
+                    )}
+                >
+                    {content ?? 'Title goes here'}
```

Comment:
```suggestion
                    {content || 'Title goes here'}
```
We might need to think about empty string